logic/src/jvmMain/kotlin/com/wire/kalium/logic/network/NetworkStateObserverImpl.kt

```diff
@@ -0,0 +1,27 @@
+/*
+ * Wire
+ * Copyright (C) 2023 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ */
+package com.wire.kalium.logic.network
+
+import kotlinx.coroutines.flow.Flow
+
+actual class NetworkStateObserverImpl : NetworkStateObserver {
+
+    override fun observeNetworkState(): Flow<NetworkState> {
+        TODO("Not yet implemented")
```

Comment:
This will most likely completely brake JVM clients, no?

On any network error, `CLI Client` and `TestService` will *poof*.

Better to just treat as always connected with internet then leaving the TODO here.